modules/01 JavaScript Programming/Assignments/20/answer20.js

```diff
@@ -0,0 +1,12 @@
+var arr = [1, 2, 5, 7];
+
+function myFilter(arr, fnc) {
+
+        return arr.filter(fnc)
```

Comment:
If we use the original "filter" function, we don't need "myFilter".
Please re-write your solution without using the original "filter" and email it to me